### PR TITLE
Master clean logisitic code whe

### PR DESCRIPTION
**Pull request by Whenrow**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fmdl** [comment]
Idea to clean :
In purchase order line `product_qty` and `product_uom_qty` have reverse logic than in other place (SM, SML, SOL, ...).

image

**Comment by amoyaux** [comment]
robodoo r+ rebase-ff

File: addons/stock/models/stock_location.py [hunk 1]
```diff
@@ -356,8 +356,8 @@ def _check_package_storage(self, product, package):
         return True
 
 
-class Route(models.Model):
-    _name = 'stock.location.route'
+class StockRoute(models.Model):
+    _name = 'stock.route'
```
Comment (Feyensv) [inline addons/stock/models/stock_location.py:360]: Could be a good opportunity to move the `stock.route` code in a specific file? :)

**Comment by Whenrow** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**